example.js

```diff
@@ -0,0 +1,57 @@
+/* function greaterThanFive(num) {
+  return num > 5
+}
+
+function isEven(num) {
+  return num % 2 === 0
+}
+
+function whatDoIDo1(arr, pred) {
+  let returnVal = true
+
+  arr.forEach((item) => {
+    if (!pred(item)) {
+      returnVal = false
+    }
+  })
+
+  return returnVal
+}
+console.log(whatDoIDo1([1,2,3], isEven))
+console.log(whatDoIDo1([], isEven))
+*/
+
+/* function whatDoIDo2Internal(remainingArr, resultArr) {
+  if (remainingArr.length < 1) {
+    return resultArr
+  } else {
+    const nextItem = remainingArr.pop()
+
+    resultArr.unshift(nextItem * 2)
+
+    return whatDoIDo2Internal(remainingArr, resultArr)
+  }
+}
+
+function whatDoIDo2(arr) {
+  return whatDoIDo2Internal(arr, [])
+}
+console.log(whatDoIDo2([5,6,4,7,3,8]))
+console.log(whatDoIDo2(['a','b','c','d']))
+*/
+
+function whatDoIDo3(arr) {
```

Comment:
why is this file part of this projecct? commited accidently?

wont take off points but be careful when pushing files not really needed.